package.json

```diff
@@ -20,7 +20,7 @@
     "lint:standard": "standard | snazzy",
     "lint:typescript": "eslint -c types/.eslintrc.json types/**/*.d.ts test/types/**/*.test-d.ts",
     "prepublishOnly": "cross-env PREPUBLISH=true tap --no-check-coverage test/build/**.test.js && npm run test:validator:integrity",
-    "test": "npm run lint && cross-env NODE_OPTIONS=no-network-family-autoselection npm run unit && npm run test:typescript",
+    "test": "npm run lint && npm run unit && npm run test:typescript",
```

Comment:
WE could remove `cross-env` from the dev-dep too

Reply:
no it's still use for prepublish